x-pack/plugins/gis/public/shared/layers/sources/ems_tms_source.js

```diff
@@ -107,6 +107,22 @@ export class EMSTMSSource extends TMSSource {
     return this._descriptor.id;
   }
 
+  async getAttributions() {
```

Comment:
How about a unit test for this method. Hard to see what goes in and what goes out